__tests__/fixtures/list/dev-deps-prod/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "dependencies": {
+    "left-pad": "^1.1.3"
```

Comment:
Try this: https://github.com/yarnpkg/yarn/tree/master/__tests__/fixtures/install/install-file-relative ;)